docs/source/devenv-install.md

```diff
@@ -7,23 +7,28 @@ the RPM packages provided by the [Copr](https://copr.fedorainfracloud.org/coprs/
 build system, which automatically builds packages with every commit merged into master.
 Packages are built for EPEL and Fedora.
 
-```shell
-    yum install -y yum-utils
-    yum-config-manager --add-repo https://copr.fedorainfracloud.org/coprs/g/leapp/leapp-devel/repo/epel-7/group_leapp-leapp-devel-epel-7.repo
-```
+* On CentOS/RHEL:
+  ```shell
+  $ yum install -y yum-utils
```

Comment:
In all cases, you are supposed to be root, not a user.
```suggestion
  # yum install -y yum-utils
```

Please fix the others as well.

Reply:
I fixed it for a few other command where it makes sense.